crates/sui-framework/tests/bag_tests.move

```diff
@@ -3,167 +3,165 @@
 
 #[test_only]
 module sui::bag_tests {
+    use sui::bag::{Self, add, contains_with_type, borrow, borrow_mut, remove};
```

Comment:
fwiw I was pushing to make this the new formatting, though I suppose this works too. 

I will just fight this again when we eventually write an autoformatter ;)

Reply:
sounds good!